guide/english/computer-science/data-structures/graphs/index.md

```diff
@@ -2,7 +2,7 @@
 title: Graphs
 ---
 ## Graphs
-A graph is a data-structure you can use to solve routing problems, such as "Are these two components connected?" and "What is the shortest path from point a to b?"
+A graph is a data structure you can use to solve routing problems, such as "Are these two components connected?" and "What is the shortest path from point a to point b?"
```

Comment:
As a suggestion, we don't know about the points a or b previously in this guide article, so to increase readability we could change the line `from point a to point b` to `from a point a to a point b` which makes the user understand that we are talking about any two points here.

Reply:
@co16353sidak I believe your suggestion of adding two extra `a`s makes it even more confusing.  I think it is fine as is for now.